homeassistant/components/google_tasks/__init__.py

```diff
@@ -31,7 +34,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except ClientError as err:
```

Comment:
Want to catch an authentication exception here or is the plan to hook in the flow in a follow up?